airbyte-bootloader/src/main/resources/application.yml

```diff
@@ -0,0 +1,115 @@
+micronaut:
+  application:
+    name: airbyte-bootloader
+  server:
+    port: 9002
+
+airbyte:
+  bootloader:
+    migration-baseline-version: ${BOOTLOADER_MIGRATION_BASELINE_VERSION}
+  flyway:
+    configs:
+      initialization-timeout-ms: ${CONFIGS_DATABASE_INITIALIZATION_TIMEOUT_MS:60000}
+    jobs:
+      initialization-timeout-ms: ${JOBS_DATABASE_INITIALIZATION_TIMEOUT_MS:60000}
+  protocol:
+    auto-upgrade-connectors: ${AUTO_UPGRADE_CONNECTORS_PROTOCOL:false}
```

Comment:
I'd move this to the `airbyte.bootloader`, also, this flag might be a bit misleading currently.
Our default behavior is to auto upgrade connectors that are not used in a sync. Cloud has a post-load function override to be able to force upgrade connectors all connectors. When I see `auto-upgrade`, I do not know which behavior it ties to.
We may want to address this when/if we rationalize those behaviors.